clang/test/Driver/sycl-unsupported.cpp

```diff
@@ -35,6 +35,10 @@
 // RUN:    -check-prefixes=UNSUPPORTED_OPT_DIAG,UNSUPPORTED_OPT
 // RUN: %clangxx -fsycl -forder-file-instrumentation -### %s 2>&1 \
 // RUN:  | FileCheck %s -DARCH=spir64 -DOPT=-forder-file-instrumentation
+// RUN: %clangxx -fsycl --coverage -### %s 2>&1 \
```

Comment:
Does this restriction apply for `%clang_cl` too?

Reply:
Yes.  I have updated the test.